README.rst

```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/Link-Document-blue.svg
+      :target: https://s3-us-west-2.amazonaws.com/login-gov-doc/login_analytics/index.html
+
+.. image:: https://img.shields.io/badge/Link-API-blue.svg
+      :target: https://s3-us-west-2.amazonaws.com/login-gov-doc/login_analytics/py-modindex.html
+
+.. image:: https://img.shields.io/badge/Link-GitHub-blue.svg
+      :target: https://github.com/18f/identity-analytics-etl
+
+.. image:: https://img.shields.io/badge/Link-Submit_Issue-blue.svg
+      :target: https://github.com/18f/identity-analytics-etl/issues
+
+.. image:: https://img.shields.io/badge/Link-Request_Feature-blue.svg
+      :target: https://github.com/18f/identity-analytics-etl/issues
+
+
+Welcome to ``login_analytics`` Documentation
+==============================================================================
+
+https://login.gov analytics code repo. Source code for:
+
+- Data ETL Pipeline
+- Analytics Query
+- Machine Learning Model
+
+Notes on 2018-11-07:
+
+    This project used to be the source code for `Redshift ETL pipeline <https://github.com/18F/identity-analytics-etl/tree/master/legacy_code>`_. We need a redesign to have a all-in-one solution for managing all kinds of analytics related work. The ETL pipeline will be an sub module of this project.
+
+
+Local development
+------------------------------------------------------------------------------
+Since AWS lambda only supports Python3.6 and Python2.7, so we use Python3.6 for development.
+
+If it is your new Mac laptop, run this to set up your development environment::
+
+    $ bash ./bin/setup-mac.sh
```

Comment:
Instead of a separate script that duplicates part of the laptop script's functionality, how about pointing people to run the laptop script, and then to install a specific version of Python, they can run `pyenv install 3.6.2`?

Reply:
I agree, I checked the the laptop scripts, it use HomeBrew to install python. And Homebrew has only Python2.7 and the latest Python3 version, which is 3.7 now. Homebrew has no control to install specific version of Python like Python3.4, 3.5, 3.6 PYPY. And it doesn't have control to enable differentiate multiple python. I recommend to use pyenv for Python development on Mac. I can help with laptop scripts.

Reply:
But I assume it takes time to update laptop scripts, and this `setup-mac.sh` is an optional thing, just leave it as is for now and remove after we update your laptop script, how about that? @monfresh

Reply:
Basically, in order for someone to be able to install Python 3.6.2, they need to be able to run `pyenv install 3.6.2`. If someone doesn't already have `pyenv`, they can run the `laptop` script, which is the standard script every TTS employee runs. Once they run the `laptop` script, they would then be able to run `pyenv install 3.6.2`. I don't think we need to maintain a separate script here.